docs/pages/access-controls/login-rules/guide.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Set Up Login Rules (Preview)
+description: Set up Login Rules to transform user traits
+---
+
+<Admonition type="warning" title="Preview">
+  Login Rules are currently in Preview mode.
+</Admonition>
+
+This guide will walk you through the process of writing, testing, and adding the
+first Login Rule to your Teleport cluster.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+Before you get started you’ll need a running Teleport Enterprise or Cloud
+cluster on version 11.3.0 or greater.
+
+Login Rules only have an effect when an SSO user logs in, so make sure you have
+configured an OIDC, SAML, or GitHub connector before you begin.
+Check the [Single Sign-On](../sso/) docs to learn how to set this up.
+
+## Step 1/4. Draft your Login Rule resource
+
+The following example will give all users a new `logins` trait set to the value
+of their current `username` trait converted to lowercase.
+Copy this example rule to a file called `my_rule.yaml` to continue with the
+guide.
+
+```
+# my_rule.yaml
+kind: login_rule
+version: v1
+metadata:
+  # Each Login Rule must have a unique name within the cluster.
+  name: my_rule
+
+  # expires is optional and usually should not be set for deployed login
+  # rules, but it can be useful to set an expiry a short time in the future
+  # while testing new Login Rules to prevent potentially locking yourself out of
+  # your Teleport cluster.
+  # expires: "2023-01-31T00:00:00-00:00"
+spec:
+  # priority orders the evaluation of Login Rules if multiple are present in the
+  # cluster, lower priorities are evaluated first.
+  priority: 0
```

Comment:
Can priority be negative or is `0` the highest priority?

Reply:
it can be negative (it's an int32). I added a mention of the default value and supported range for the reference page, I think it can be left out of the guide